torch/_inductor/lowering.py

```diff
@@ -1999,21 +1999,29 @@ def inner_fn(index):
 
 @register_lowering(aten.bucketize, type_promotion_kind=None)
 def bucketize(
-    input: TensorBox,
+    input: Union[TensorBox, float, int],
     boundaries: TensorBox,
     *,
     out_int32: bool = False,
     right: bool = False,
 ):
     assert len(boundaries.get_size()) == 1
 
+    # First check if input is not a tensor in which case fallback to
+    # aten.bucketize.Scalar
+    if isinstance(input, Number):
+        return fallback_handler(aten.bucketize.Scalar, add_to_fallback_set=False)(
+            input, boundaries, out_int32=out_int32, right=right
+            )
+
```

Comment:
Since we wrap scalars as tensors in the decomp, this path should never be hit

Reply:
Right, fixed.